test/asyncTasks.test.ts

```diff
@@ -0,0 +1,38 @@
+import assert = require('assert')
+import { Action, ActorWorld } from '../src'
+import type { IActorWorldOptions } from '../src/ActorWorld'
+import asyncAssignTasks from '../src/asyncAssignTasks'
+
+describe(asyncAssignTasks.name, () => {
+  const options: IActorWorldOptions = { parameters: {} } as IActorWorldOptions
+
+  // success
+  it('successfully loads a task', async () => {
+    class TestWorld extends ActorWorld {
+      public goodtask: () => Action<void>
+    }
+
+    const world = new TestWorld(options)
+
+    // Load the task
+    await asyncAssignTasks(world, `${__dirname}/fixtures/tasks/goodtasks`)
```

Comment:
I don't understand something here.

On line 12 we've defined `goodtask` as `() => Action<void>` but that's also what's defined in the `goodtask.ts` fixture.

Should we be using the result of this call to `asynAssignTasks` to set up the `goodtask` property on our `TestWorld` on line 12, instead of hard-coding / duplicating it?